bcrs-ui/src/main.ts

```diff
@@ -0,0 +1,37 @@
+// Vue Libraries
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import router from './router'
+import { store } from '@/store'
+
+// Styles
+import '@/assets/styles/base.scss'
+import '@/assets/styles/layout.scss'
+import '@/assets/styles/overrides.scss'
+
+// Base App
+import App from './App.vue'
+
+// Helpers
+import { fetchConfig } from '@/utils'
+
+Vue.config.productionTip = false
+Vue.use(Vuetify)
+const vuetify = new Vuetify()
+
+/**
+ * Fetch config from server, then load Vue
+ */
+fetchConfig()
+  .then(() => {
+    new Vue({
```

Comment:
See latest code in coops-ui as I think we'll need all of it here also.

Reply:
Will do, i will include the auth alongside your new refresh logic 👍